src/components/info-reveal/readme.md

```diff
@@ -0,0 +1,33 @@
+# z-info-reveal
+
+Interactive info panel used to show short sentences, one at a time. Might contain a label and a custom icon on initial, closed state.
```

Comment:
Suggerirei per la seconda parte: "The initial (closed) state always displays a custom icon and may contain a text label."